HACKING.md

```diff
@@ -42,6 +42,14 @@ Add `$GOPATH/bin` to your `PATH`, so you can run the go programs you install:
 your `$GOPATH` is more complex than a single entry you'll need to adjust the
 above).
 
+Note that if you are using go 1.16 or newer you need to disable the
+go modules feature. Use:
```

Comment:
... disable the go modules feature by setting `GO111MODULE=off` in your environment?

Reply:
Keeping it as it is makes it stand out better, on the other hand.